docs/installation_guide.rst

```diff
@@ -313,7 +313,16 @@ After the app is built, if everything went well you should see the below text in
 
 If you get any errors about missing dependencies, try to install them via npm and run again; consult the troubleshooting section for other errors.
 
-3. Alternatively, you can also choose to build the application. In order to do so, navigate to your rotki development directory and execute the ``package.bat`` file. NOTE: You will need to edit the directory in Line 30 to the name of your rotki development directory.
+3. Alternatively, you can also choose to build the application. In order to do so, navigate to your rotki development directory and execute the ``package.bat`` file. NOTE: You will need to edit the directories in the batch file to point to where you built pysqlcipher3 and your rotki development directory. The relevant section of the script (`example <https://github.com/isidorosp/rotki/blob/47c92cdb0b48115fb668efe59219783efeb075ce/package.bat#L27-L30>`_) is::
```

Comment:
Hey can you point to the upstream develop commit though? Not your fork? Also since the link is there, there is no need to show the code.

Reply:
I think I got the right one now.